packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -210,7 +214,6 @@ export async function startTransformationJob(uploadId: string) {
         }
         throw new Error('Start job failed')
     }
-    transformByQState.setJobId(encodeHTML(jobId))
```

Comment:
why remove jobId from transform state? This can cause some metrics to miss jobId?

Reply:
Oh I didn't remove, I just moved it to happen sooner. I set the jobId right after the upload completes so that it is populated in the job history tab sooner.